perfkitbenchmarker/scripts/object_storage_api_tests.py

```diff
@@ -67,6 +72,21 @@
                      'particular test scenario. Currently only applicable to '
                      'the ListConsistency scenario, ignored in others.')
 
+flags.DEFINE_integer('objects_per_stream', 1000, 'The number of objects to '
```

Comment:
Same comments to these flags as I made in the pkb "driver" script. 

I think we should be careful about module layering here. We need to think (clearly) about how we plan to support object size distribution and number of streams. Even though we don't have the object size distribution support in this PR, it's going to be helpful to the (very) near future when you add such support.

My thought (open for discussions):
1. We should add object size distribution support to this file here, it will take a distribution config file (# of streams, objects per stream, and size distribution), or a set of flags specifying the distribution (per design doc), and work off on that. 
2. The "driver" script will either pass through user's test configure file straightly to this file (after some sanity check), or create a config file by itself for the api_multi_stream scenario based on the 3 flags user passed-in for that scenario.

If that's the direction we want to head for, then we should change the flags here (especially the name) to reflect the end state when they are used to describe a distribution. We don't have to implement the full distribution here in this PR, (e.g., we can assume 100% object will be of the same size). But we do want to get the flag names (and meaning) correct so we don't need to change the flags next time when we add distribution support.